crates/ika-protocol-config/src/lib.rs

```diff
@@ -362,6 +374,48 @@ impl ProtocolConfig {
             // responsibility is shared amongst more nodes. We can increase that once we do have
             // higher confidence.
             consensus_bad_nodes_stake_threshold: Some(20),
+            consensus_rounds_delay_per_mpc_protocol: HashMap::from([
+                (
+                    DWALLET_DECRYPTION_KEY_RESHARE_REQUEST_EVENT_NAME.to_string(),
+                    HashMap::from([]),
+                ),
+                (
+                    FUTURE_SIGN_REQUEST_EVENT_NAME.to_string(),
+                    HashMap::from([]),
+                ),
+                (
+                    ENCRYPTED_SHARE_VERIFICATION_REQUEST_EVENT_NAME.to_string(),
+                    HashMap::from([]),
+                ),
+                (
+                    START_NETWORK_DKG_EVENT_STRUCT_NAME.to_string(),
+                    HashMap::from([]),
```

Comment:
In round 3, wait for 20 more rounds before advancing (i.e. when you wait for messages from round 2.)